### DISCUSSION TITLE
Fair Value still the same in the --free-version

### DISCUSSION THREAD

**Discussion by PTemp1e** [ctubio/Krypto-trading-bot#1132] [General]
Hallo,
thanks for this project. The Bot is running and how i read in the manual the Fair Value should update every 121 seconds, but in the Free Version its still the same when you Start the bot. Is this normal??

**Comment by ctubio** [comment]
good morniiing''
if there is market movements the fairvalue should be updated consequently;

:koala: what exchange and currency pairs are you using that are not updating?

**Comment by PTemp1e** [comment]
The exchange is KuCoin and i testet it with 5 or 6 pairs. Everytime the same. I start the bot, he gets his Fair Value at the beginning (under Stats left corner down the Yellow line) and never change it. Under Takers i see the orderbook on the right.

**Reply by ctubio** [reply]
:koala: mmm im not able to reproduce this, on Kucoin BTC/USDT the fairvalue updates as expected as far as i can see.

consider t share what currency pairs are you using (so we can test the same) or run the bot with --debug argument to see if there is any interesting msg for your kucoin account

**Reply by PTemp1e** [reply]
The currency pair f.e. is XRP/USDT and BTC/USDT. With the --debug comment i can't see any errors. Could it be a reason that i use a Subaccount on Kucoin??
I will test an other Raspberry (4B / 4GB) if there is the same or change the excange. Thanks for your help

**Reply by PTemp1e** [reply]
The Output for the argument --debug-quotes doesn't change all the time. Is this maybe the failure.
